PowerEditor/src/Notepad_plus.cpp

```diff
@@ -1351,46 +1351,29 @@ void Notepad_plus::removeEmptyLine(bool isBlankContained)
 	_findReplaceDlg.processAll(ProcessReplaceAll, &env, true);
 }
 
-void Notepad_plus::getMatchedFileNames(const TCHAR *dir, const vector<generic_string> & patterns, vector<generic_string> & fileNames, bool isRecursive, bool isInHiddenDir)
+void Notepad_plus::getMatchedFileNames(const TCHAR *dir, const vector<generic_string> & patterns, vector<generic_string> & fileNames, bool isRecursive, bool isInHiddenDir, const Progress* progress)

```

Comment:
It looks like this is an optional parameter? Maybe it'd be good to declare it as `_In_opt_`, for documentation and clarity?

i.e.:
`_In_opt_ const Progress* const progress` or `_In_opt_ const Progress* progress`

....depending on your own level `const` naziism